### DISCUSSION TITLE
Capture video frame

### DISCUSSION THREAD

**Discussion by nilmarpublio** [quasarframework/quasar#11848] [General - Components / Directives / etc] [answered]
Hello everyone,
I'm starting with quasar framework, and I'm finding it difficult to capture a frame of any video every two seconds and display this captured frame .

Can someone help me ?
Thanks in advance.

**Comment by MilosPaunovic** [comment]
Please create a Codepen reproduction of your particular issue so you could get better support, or at least share a code sample of some kind so someone could understand what exactly are you trying to accomplish.

**Comment by hawkeye64** [comment]
Also would be helpful is knowing how you are receiving your video feed.
Do you have a server and proxy it to the browser?
Are you getting the video direct from some other server and putting it into a `<video>`?
It's extremely helpful when asking for help to be thorough in what you have tried and what's not working. Plus, your environment and such.

Also, this is not really a Quasar issue, but there's a lot of people in the community still willing to help. So, providing enough information is key to getting a response.

**Comment by nilmarpublio** [comment]
The video is a .mp4 file inside  the public folder.

The proposal is that I can capture a frame of this video every 2 seconds to identify and mark objects to show(carrousel, card... Whatever it is)
The goal is to reduce processing time by at least 48 frames between each capture.

**Comment by hawkeye64** [comment, marked as answer]
You can do something like this. Just change it up a bit to work with Vue (refs, etc), if needed

```js
// pass reference to the video element
function generateThumbnail (video) {     
    //generate thumbnail URL data
    const context = thecanvas.getContext('2d')
    context.drawImage(video, 0, 0, 220, 150)
    const dataURL = thecanvas.toDataURL()

    //create img
    const img = document.createElement('img')
    img.setAttribute('src', dataURL)

    //append img in container div
    return img
}
```

**Reply by nilmarpublio** [reply]
Thanks a lot.

**Reply by hawkeye64** [reply]
Note, this returns an img tag, not the actual image.
